src/components/AdminModal.vue

```diff
@@ -485,40 +574,10 @@ export default defineComponent({
         }
       }
     },
-    // handleFileChange(event: any) {
-    //   const file = event.target.files[0]
-    //   if (file && /\.(jpg|jpeg|png)$/i.test(file.name)) {
-    //     const reader = new FileReader()
-    //     reader.onload = (e) => {
-    //       // Remove the data URL prefix to get just the base64 string
-    //       if (e.target != null && typeof e.target.result == 'string') {
-    //         this.selectedPhoto = e.target.result
-    //         this.photo = e.target.result.split(',')[1]
-    //       }
-    //     }
-    //     reader.readAsDataURL(file)
-    //     console.log(this.selectedPhoto)
-    //     console.log(this.photo)
-    //   } else {
-    //     // Reset selectedPhoto or show error message
-    //     this.selectedPhoto = ''
-    //     alert('Please select a JPEG, JPG, or PNG file.')
-    //   }
-    // },
-    async handleImageUpload(event: any) {
```

Comment:
I added handleImageUpload to replace handleFileChange in #73, do undo this change